jobs/WebUi/BuildAndPublish.groovy

```diff
@@ -0,0 +1,57 @@
+def buildPackage(String repo_dir){

```

Comment:
sounds like ```repo_dir``` you have designed/prepared for separate those code out of on-build-config :-)